### DISCUSSION TITLE
Gamepad API hooks?

### DISCUSSION THREAD

**Discussion by joshuacurtiss** [replit/kaboom#166] [Ideas]
Your functions that simplify keyboard/mouse actions are nice for extremely simple control coding. Amazingly simple.  Have you thought of incorporating similar hooks to simplify gamepad support? ([Using the Gamepad API](https://developer.mozilla.org/en-US/docs/Web/API/Gamepad_API/Using_the_Gamepad_API))  I'll probably be implementing gamepad support manually but thought that it may be something worth creating a simplified interface like you've accomplished with keyboard/mouse.

Just struck me today and figured I'd bounce it off you.

**Comment by slmjkdbtl** [comment]
Definitely will have that in the lib, adding to [todo](https://github.com/replit/kaboom/projects/1#card-67808695)

**Comment by slmjkdbtl** [comment]
Gamepads are supported in v3000 beta
